internal/util/update/update_test.go

```diff
@@ -34,7 +34,9 @@ import (
 )
 
 const (
-	kptRepo = "github.com/GoogleContainerTools/kpt"
+	kptRepo         = "github.com/GoogleContainerTools/kpt"
+	mainBranch      = "master"
```

Comment:
We should call this `masterBranch` instead of `mainBranch`. No branch is special, but most of the tests uses a mix of `main` and `master` as the name. This name would be a little confusing.

Reply:
Renamed.